weave/trace/weave_client.py

```diff
@@ -560,9 +560,13 @@ def create_call(
             op_str = op
 
         inputs_redacted = redact_sensitive_keys(inputs)
+        if op.postprocess_inputs:
+            inputs_postprocessed = op.postprocess_inputs(inputs_redacted)
```

Comment:
I'm wondering whether it would be useful to have `postprocess_inputs` work on the `inputs` rather than `inputs_redacted` - that way, if we were redacting something the user didn't want redacted they would have a way to opt out. But our current redaction is pretty limited and I doubt there are many false positives, so this is probably for the best. Just raising it for consideration.

Reply:
Maybe redacting could be part of postprocess.  We could let the user pass a list of funcs?

Reply:
Thinking about this more, I think the cleanest would be to keep it how you have it, and then if we later discover that people want to turn off redaction we could  introduce a separate `redact` boolean keyword arg on the opt that defaults to True.